vg/build/Dockerfile

```diff
@@ -1,24 +1,25 @@
-FROM ubuntu:14.04
+FROM ubuntu:15.10
 
 MAINTAINER Charles Markello, cmarkell@ucsc.edu
 
 # Install vg dependencies
-RUN apt-get update \
-	&& apt-get update && apt-get install -y \
-	build-essential \
-	gcc-4.9 \
-	g++-4.9 \
-	pkg-config \
-	sudo \
-	git \
-	make \
-	protobuf-compiler \
-	libprotoc-dev \
-	libjansson-dev \
-	libbz2-dev \
-	libncurses5-dev \
-	automake libtool jq samtools curl unzip redland-utils \
-	librdf-dev cmake pkg-config wget bc gtk-doc-tools raptor2-utils rasqal-utils bison flex
+RUN apt-get update && \
+	apt-get update &&\
```

Comment:
BTW, I prefer having a continued line start with an operator, as in 

```
RUN apt-get update \
    && apt-get upgrade \
    && apt-get install -y \
        build-essential \
        …
```

The reasoning is that having the operator/separator at the beginning of the continuing line creates only two sets of lines. The first line is one set, the subsequent lines form the second set. With the separator at the end of a continued line, there are three distinct sets: 1) the first line, 2) the last line and 3) the lines in between. This makes appending lines in future commits more error prone and creates larger diffs because the old last line needs to be changed.